docs/en/reference/Properties-List.md

```diff
@@ -8,6 +8,8 @@ priority: 0
 * Table of Contents
 {:toc}
 
+TODO: Ensure the categories are up to date (do we need to add SERVER?)
```

Comment:
Ideally, I would like to update properties to better reflect these categories. However, for now, it looks like "common" is supposed to behave like "server".

Reply:
Do you know if there is anything preventing us from renaming `common` to `server`?

Reply:
I think there are configurations in `common` that get used by the client too.

One thing that could improve the categorization is to incorporate `PropertyKey` scopes into `ConfDocGenerator`